diff --git a/admin/views/view.php b/admin/views/view.php
index fffb3b0ed..a6ff9d2cb 100644
--- a/admin/views/view.php
+++ b/admin/views/view.php
@@ -480,6 +480,7 @@ PT_CV_Functions::view_submit();
 						),
 					),
 				);
+		                $options = apply_filters(PT_CV_PREFIX_ . 'filter_settings_final', $options);
 				echo PT_Options_Framework::do_settings( $options, $settings );
 				?>
 			</div>
