Index: admin/views/view.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/views/view.php	(date 1570159575000)
+++ admin/views/view.php	(date 1570659804000)
@@ -466,6 +466,7 @@
 						),
 					),
 				);
+				$options = apply_filters(PT_CV_PREFIX_ . 'filter_settings_final', $options);
 				echo PT_Options_Framework::do_settings( $options, $settings );
 				?>
 			</div>
